src/Client/Extensions/HttpClientPremiumInfoExtensions.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Brock Allen & Dominick Baier. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using System;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using IdentityModel.Internal;
+
+namespace IdentityModel.Client
+{
+    /// <summary>
+    /// HttpClient extensions for OIDC premiuminfo
+    /// </summary>
+    public static class HttpClientPremiumInfoExtensions
```

Comment:
OIDC does NOT specifies anything like "premiuminfo" - there's only userinfo endpoint.